### PR TITLE
Rspec tests

### PR DESCRIPTION
**Pull request by joseluispodemos**

### CODE REVIEW

File: app/models/amendments/articulated_amendment.rb [hunk 1]
```diff
@@ -12,9 +12,5 @@ def articulated?
     def allow_articulated?
       true
     end
-
-    def section
```
Comment (leio10) [inline app/models/amendments/articulated_amendment.rb]: Why does the section is not needed anymore?

File: app/models/concerns/has_modification_section.rb [hunk 2]
```diff
@@ -8,10 +8,6 @@ module HasModificationSection
   end
 
   def first_modification
-    modifications.first
-  end
-
-  def section
-    @section ||= first_modification.section if modifications?
+    modifications&.first
```
Comment (leio10) [inline app/models/concerns/has_modification_section.rb]: can `modifications` be `nil`?

File: app/helpers/amendments_helper.rb [hunk 3]
```diff
@@ -4,11 +4,11 @@ module AmendmentsHelper
   def collection_title(collection)
     return t('helpers.views.empty_collection', model: collection.model_name.human(count: 0).downcase) if collection.empty?
 
-    content_tag(:b, t('helpers.views.choose_model', model: collection.model_name.human.downcase)) + ':<br>'.html_safe
+    safe_join(content_tag(:b, t('helpers.views.choose_model', model: collection.model_name.human.downcase)), ':<br>')
```
Comment (leio10) [inline app/helpers/amendments_helper.rb]: You should fix this `safe_join` call as the others

File: app/helpers/amendments_helper.rb [hunk 4]
```diff
@@ -4,11 +4,11 @@ module AmendmentsHelper
...
   end
 
   def browse_title(object)
-    content_tag(:b, object.class.model_name.human) + ": #{object.full_title}" + '<br>'.html_safe
+    safe_join(content_tag(:b, object.class.model_name.human + ': '), content_tag(:span, object.full_title), '<br>')
```
Comment (leio10) [inline app/helpers/amendments_helper.rb]: You should fix this `safe_join` call as the others.

File: app/models/amendments/articulated_amendment.rb (hunk 1 above)
Comment (joseluispodemos) [reply app/models/amendments/articulated_amendment.rb]: Beacuse of this https://github.com/podemos-info/national_budgets/blob/61be6fee814689f8b240546996786f641d6f0dc9/app/models/amendments/articulated_amendment.rb#L6

File: app/models/concerns/has_modification_section.rb (hunk 2 above)
Comment (joseluispodemos) [reply app/models/concerns/has_modification_section.rb]: No, it shouldn't.
```
2.6.3 :003 > Amendments::StandardAmendment.new.modifications
 => #<ActiveRecord::Associations::CollectionProxy []>
```
Fixed here: https://github.com/podemos-info/national_budgets/commit/96b5681b9e4ddc61e0f5f1d907da195abdbf5dda

File: app/helpers/amendments_helper.rb (hunk 4 above)
Comment (joseluispodemos) [reply app/helpers/amendments_helper.rb]: Fixed https://github.com/podemos-info/national_budgets/commit/8039afc8aa6e7e194df3b2d7823864dbced6a462

**Comment by joseluispodemos** [comment]
> 1st_place_medal great work! grin

Thanks! :blush:

**Pull request merged**